nomad/structs/csi.go

```diff
@@ -372,6 +380,8 @@ func NewCSIVolume(volumeID string, index uint64) *CSIVolume {
 		ID:          volumeID,
 		CreateIndex: index,
 		ModifyIndex: index,
+		CreateTime:  time.Now().UnixNano(),
+		ModifyTime:  time.Now().UnixNano(),
```

Comment:
These values will be different because we're calling it twice (plus now we have to make two vDSO syscalls). Given all the callers for this function are tests, that might make spurious test failures in the future. Maybe this function should take a `now` parameter as we did for `NewDeployment`?